exercises/fizz-buzz/README.md

```diff
@@ -21,9 +21,7 @@ Inside of RStudio, simply execute the `test_<exercise_name>.R` script. This can
 
 "Fizz-Buzz" originated as a children's game that helped children learn to divide. It is now regularly used in coding interviews. 
 
-1. [Rosetta Code](https://rosettacode.org/wiki/FizzBuzz) 
-2. [Code Golf](https://code-golf.io/fizz-buzz)
-
+1. [Actual Source](https://imranontech.com/2007/01/24/using-fizzbuzz-to-find-developers-who-grok-coding/D)
```

Comment:
The `D` at the end redirects my browser to https://imranontech.com/2007/02/20/did-bill-gates-say-the-640k-line/ somehow. Please fix & also revert the deletions of `1.` & `2.`.

Reply:
Yes done. You can do the squash for me please. I don't wanna break things.